File: js/http-intro/handler.js
exports.httpIntro = async event => {
  let body = 'Hello World';

  if (event.queryStringParameters && event.queryStringParameters.name) {
    body = `Hi ${event.queryStringParameters.name}`;
  }

  return {
    statusCode: 200,
    headers: {},
    body
  }
}

File: python/http-intro/handler.py
def httpIntro(event, context):
    body = 'Hello World'

    if 'queryStringParameters' in event and 'name' in event['queryStringParameters']:
        body = 'Hi ' + event['queryStringParameters']['name']

    return {
        'statusCode': 200,
        'headers': {},
        'body': body
    }
